scripts/auth_init.ps1

```diff
@@ -0,0 +1,9 @@
+. ./scripts/loadenv.ps1
+
+$venvPythonPath = "./.venv/scripts/python.exe"
```

Comment:
this path give me an error I replaced it with `"./scripts/.venv/scripts/python.exe"` and `"./scripts/.venv/bin/python"` in line 6